modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js

```diff
@@ -128,11 +134,9 @@ export default class Blogs {
 			'.cover-image-caption'
 		);
 
-		Liferay.on('coverImageDeleted', this._removeCaption, this);
-		Liferay.on(
-			['coverImageUploaded', 'coverImageSelected'],
-			this._showCaption,
-			this
+		this._imageSelectorCoverImageSubscription = State.subscribe(
```

Comment:
`State.subscribe` is how you get notified of changes to global state. In a React component, you would use `useLiferayState()` as a convenience instead. You can see an example of that further down.